tests/conftest.py

```diff
@@ -856,3 +858,76 @@ def example_psfex_psf_files():
     if not ( psfpath.is_file() and psfxmlpath.is_file() ):
         raise FileNotFoundError( f"Can't read at least one of {psfpath}, {psfxmlpath}" )
     return psfpath, psfxmlpath
+
+
+@pytest.fixture
+def ptf_image_new():
+    datadir = os.path.join(FileOnDiskMixin.local_path, 'test_data/PTF_examples')
+    if not os.path.isdir(datadir):
+        os.makedirs(datadir)
+
+    filename = 'PTF201104234316_2_o_44887_11.w.fits'
+
+    path = os.path.join(datadir, filename)
+    cachedpath = filename + '_cached'
+
+    if os.path.isfile(cachedpath):
+        _logger.info(f"{path} exists, not redownloading.")
+    else:
+        url = f'https://portal.nersc.gov/project/m2218/pipeline/dec+04/ccd_11/{filename}'
+        retry_download(url, cachedpath)  # make the cached copy
+
+    shutil.copy2(cachedpath, path)  # make a temporary copy to be deleted at end of test
+    prov = provenance_base
```

Comment:
Is there enough code overlap between this and DECam that it'd be worth breaking some of it out into a function?  (I'm guessing probably not, the savings would just be a handful of lines.)

Reply:
I have an idea for what to do to clean up the fixtures and the caching, but that will come later today.